Templates/CSharp/Desktop/ProjectTemplates/CSharp/Test/DesktopUnitTestTemplate/MSTestV2Desktop.vstemplate

```diff
@@ -25,8 +25,8 @@
   </WizardExtension>
   <WizardData>
     <packages repository="extension" repositoryId="mstestProjectTemplate" isPreunzipped="false" >
-      <package id="MSTest.TestAdapter" version="2.2.8" skipAssemblyReferences="false"  />
-      <package id="MSTest.TestFramework" version="2.2.8" skipAssemblyReferences="false" />
+      <package id="MSTest.TestAdapter" version="2.2.10" skipAssemblyReferences="false"  />
```

Comment:
@Haplois Do we want to start using `MSTest` package right now or keep the multiple packages for now?

Reply:
I can do this in a separate PR if @Haplois deems it necessary to start using the new package. But for the current PR I'd stick with the old packages.